app.rb

```diff
@@ -0,0 +1,166 @@
+require_relative './book'
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+
+def list_books
+  puts "******* \n List of all book \n ****************"
+  puts " No books addded \n ************" if @books.length.zero?
+  @books.each do |book|
+    puts "id: #{book.id} Title: #{book.title} Author: #{book.author}"
+  end
+  puts '****************'
+end
+
+def list_people
+  puts "******* \n List of all people \n ****************"
+  puts " No people is added \n ************" if @people.length.zero?
+  @people.each do |person|
+    puts "Id: #{person.id} name: #{person.name} Age: #{person.age}"
+  end
+  puts '****************'
+end
+
+def list_rental
+  puts "******* \n List of all rental of a person \n ****************"
+  puts " No one rent a book \n ************" if @rentals.length.zero?
+  puts 'Enter id of a person'
+  person_id = gets.chomp.to_i
+
+  @rentals.each do |rental|
+    if rental.person.id == person_id
+      puts "On #{rental.date} the book:  #{rental.book.title} is rented by #{rental.person.name}"
+      puts '****************'
+    end
+  end
+end
```

Comment:
- [ ] When I'm trying to list rentals for a person, I get nothing on the screen. Please fix this issue.

![image](https://user-images.githubusercontent.com/83803414/201491098-518bf5e9-232f-4cdf-a1cd-5b3e0e185962.png)